views/default/input/autocomplete.php

```diff
@@ -35,17 +35,11 @@
 	$params['match_owner'] = $vars['match_owner'];
 	unset($vars['match_owner']);
 }
-$ac_url_params = http_build_query($params);
+$ac_url_params = urldecode(http_build_query($params));
```

Comment:
@mrclay For whatever reason it didn't work without urldecode when I tested it on the weekend. For whatever reason it now also works without...

Checking out your PR now.